src/package.xml

```diff
@@ -118,6 +118,8 @@
         <members>TDTM_TriggerScaffolds_TEST</members>
         <members>TERM_CourseOff_TDTM</members>
         <members>TERM_CourseOff_TEST</members>
+        <members>TG_ValidateData_TDTM</members>
```

Comment:
The new TDTM and test classes should be added to the package.xml, otherwise they won't deploy to new orgs.

Reply:
@robertbwatson , thank you for the heads up on this. I've updated the package.xml and it now contains COFF_CannotDelete_TDTM, COFF_CannotDelete_TEST, CCON_CannotDelete_TDTM, and CCON_CannotDelete_TEST.